app/src/main/java/org/oppia/app/home/HomeFragment.kt

```diff
@@ -12,7 +12,8 @@ import javax.inject.Inject
 
 /** Fragment that contains an introduction to the app. */
 class HomeFragment : InjectableFragment(), TopicSummaryClickListener {
-  @Inject lateinit var homeFragmentPresenter: HomeFragmentPresenter
+  @Inject
+  lateinit var homeFragmentPresenter: HomeFragmentPresenter
```

Comment:
Optional nit, but I suggest undoing this. I personally prefer keeping annotations on the same line unless they don't fit within word wrapping, or there are multiple annotations on the same field.

Reply:
Actually, I prefer the same way as mentioned by you, but the `Option+Cmd+L` rearranges this in two lines, and I missed this. 

Again, fixed as per suggestion and converted to single line code.